Makefile

```diff
@@ -1,26 +1,66 @@
+.PHONY: help venv conda docker docstyle format style black test lint check
+.DEFAULT_GOAL = help
+
+PYTHON = python
+PIP = pip
+CONDA = conda
+SHELL = bash
+
+help:
+	@printf "Usage:\n"
+	@grep -E '^[a-zA-Z_-]+:.*?# .*$$' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?# "}; {printf "\033[1;34mmake %-10s\033[0m%s\n", $$1, $$2}'
+
+conda:  # Set up a conda environment for development.
+	@printf "Creating conda environment...\n"
+	${CONDA} create --yes --name env python=3.6
+	( \
+	source activate env; \
```

Comment:
`source activate` is out of favor now. Should be `conda activate` like on line 21

Reply:
Gotcha!